app/core/download.py

```diff
@@ -249,14 +250,17 @@ def get_whole_database_dump(ingest_cycle_start: str, db=Depends(get_db)):
 def replace_slug_with_qualified_url(
     df: pd.DataFrame,
     public_app_url: str,
-    url_cols: list[str] = ["Family Slug", "Document Slug"],
+    url_cols: Optional[list[str]] = None,
```

Comment:
Not a blocker just wondering why this is better!

Reply:
Because we shouldn't be using mutable data structures for argument defaults!

Reply:
Nice!